sonar-db/src/main/java/org/sonar/db/user/GroupMembershipDao.java

```diff
@@ -74,34 +73,32 @@ public int countMembers(SqlSession session, UserMembershipQuery query) {
     return mapper(session).countMembers(params);
   }
 
-  public Map<String, Integer> countUsersByGroups(final DbSession session, Collection<Long> groupIds) {
-    final Map<String, Integer> result = Maps.newHashMap();
-    DatabaseUtils.executeLargeInputs(groupIds, new Function<List<Long>, List<GroupUserCount>>() {
-      @Override
-      public List<GroupUserCount> apply(@Nonnull List<Long> input) {
-        List<GroupUserCount> userCounts = mapper(session).countUsersByGroup(input);
-        for (GroupUserCount count : userCounts) {
-          result.put(count.groupName(), count.userCount());
-        }
-        return userCounts;
-      }
-    });
+  public Map<String, Integer> countUsersByGroups(DbSession session, Collection<Long> groupIds) {
+    Map<String, Integer> result = Maps.newHashMap();
+    executeLargeInputs(
+        groupIds,
+        input -> {
+          List<GroupUserCount> userCounts = mapper(session).countUsersByGroup(input);
+          for (GroupUserCount count : userCounts) {
+            result.put(count.groupName(), count.userCount());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)